### PR TITLE
scalaz-stream is on maven now

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: documentation/manual/releases/release24/migration24/Migration24.md [hunk 1]
```diff
@@ -43,12 +43,12 @@ sbt.version=0.13.8
...
 ### Specs2 support in a separate module
 
-If you were previously using Play's specs2 support, you now need to explicitly add a dependency on that to your project.  Additionally, specs2 now requires `scalaz-stream` which isn't available on maven central or any other repositories that sbt uses by default, so you need to add the `scalaz-stream` repository as a resolver:
+If you were previously using Play's specs2 support, you now need to explicitly add a dependency on that to your project.  Additionally, specs2 now requires `scalaz-stream` which is available on maven central so you don't need to add the `scalaz-stream` repository as a resolver:
 
 ```scala
 libraryDependencies += specs2 % Test
 
-resolvers += "scalaz-bintray" at "https://dl.bintray.com/scalaz/releases"
+// resolvers += Resolver.bintrayRepo("scalaz", "releases")
```
Comment (schmitch) [inline documentation/manual/releases/release24/migration24/Migration24.md]: there is no reason to comment that out if it's not necessary. just remove it

File: documentation/manual/releases/release24/migration24/Migration24.md [hunk 2]
```diff
@@ -43,12 +43,12 @@ sbt.version=0.13.8
 
 ### Specs2 support in a separate module
 
-If you were previously using Play's specs2 support, you now need to explicitly add a dependency on that to your project.  Additionally, specs2 now requires `scalaz-stream` which isn't available on maven central or any other repositories that sbt uses by default, so you need to add the `scalaz-stream` repository as a resolver:
+If you were previously using Play's specs2 support, you now need to explicitly add a dependency on that to your project.  Additionally, specs2 now requires `scalaz-stream` which is available on maven central so you don't need to add the `scalaz-stream` repository as a resolver:
```
Comment (schmitch) [inline documentation/manual/releases/release24/migration24/Migration24.md]: Guess it's fine to just remove the paragraph about scalaz.

Comment (graingert) [inline documentation/manual/releases/release24/migration24/Migration24.md]: well 0.7a is not on maven, only 0.7.2a

Comment (schmitch) [inline documentation/manual/releases/release24/migration24/Migration24.md]: actually as far as I know the version 3.8.4 which play 2.6 will use doesn't need scalaz-stream.

**Comment by graingert** [comment]
https://github.com/playframework/playframework/pull/6498

File: documentation/manual/releases/release24/migration24/Migration24.md (hunk 2 above)
Comment (graingert) [inline documentation/manual/releases/release24/migration24/Migration24.md]: spec2 3.6.3 uses a version of scalaz-stream that's on maven.

**Comment by graingert** [comment]
This only applies for spec2 2.6.6 or later

**Pull request merged**